prog/mutation.go

```diff
@@ -269,6 +269,117 @@ func Minimize(p0 *Prog, callIndex0 int, pred func(*Prog, int) bool) (*Prog, int)
 		p0 = p
 		callIndex0 = callIndex
 	}
+
+	triedPaths := make(map[string]bool)
+
+	var rec func(p *Prog, call *Call, arg *Arg, path string) bool
```

Comment:
This must not happen during crash minimization, because each try can take 10 minutes.